motionplan/planManager.go

```diff
@@ -508,13 +494,59 @@ func (pm *planManager) plannerSetupFromMoveRequest(
 		}
 	}
 
+	// extract inputs corresponding to the frame
+	frameInputs, err := pm.frame.mapToSlice(seedMap)
+	if err != nil {
+		return nil, err
+	}
+
+	// create robot collision entities
+	movingGeometriesInFrame, err := pm.frame.Geometries(frameInputs)
+	movingRobotGeometries := movingGeometriesInFrame.Geometries() // solver frame returns geoms in frame World
```

Comment:
This seems potentially dangerous to do before checking the error

Reply:
Yes, good catch.